tests/fips/mozilla_nss/firefox_nss.pm

```diff
@@ -58,15 +58,26 @@ sub search_certificates {
     wait_still_screen 10;
 }
 
+sub firefox_prefs {
```

Comment:
as there's already a 'firefox_preferences' function, to avoid confusion I'd prefer to join the two functions or to change this name with a more distinct one